AncientReligions/interface/coats_of_arms.txt

```diff
@@ -116,7 +116,7 @@ culture =
 		texture = {
 			file = "gfx\\coats_of_arms\\dynasties4.tga"
 			size = { x = 16 y = 4 }
-			noOfFrames = 57
+			noOfFrames = 52
 			color = 0
```

Comment:
Well, it's the number of frames in dynasties4.tga of Charlemagne CoA DLC (dlc048), and I definitely confirm it's 52 and it's the same value in vanilla.coats_of_arms.txt (but I don't think it causes any issue because the extra frames are never referenced from dynasties.txt anyway)